README.md

```diff
@@ -1,320 +1,353 @@
+-  language • 🟢 lang
+-  latitude • 🟡 lat { only with **coord** }
+-  length • 🟢 len
+-  less or equal • 🟡 le { binary operators }
+-  less than • 🟡 lt { binary operators }
+-  library • 🟢 lib
+-  linked list • 🟢 ll
+-  longitude • 🟡 lon { only with **coord** }
 
 ### M
 
--  🟢 max • maximum
--  🟢 mcu • microcontroller
--  🟢 mem • memory
--  🟡 meta • metadata {data science}
--  🟢 mid • middle
--  🟢 min • minimum
--  🟢 misc • miscellaneous
--  🟢 mng • manager
--  🟢 mod • modulo
--  🟢 msg • message
--  🟡 mut • mutable {variables}
+-  manager • 🟢 mng
+-  maximum • 🟢 max
+-  memory • 🟢 mem
+-  message • 🟢 msg
+-  metadata • 🟡 meta { data science }
+-  microcontroller • 🟢 mcu
+-  middle • 🟢 mid
+-  minimum • 🟢 min
+-  miscellaneous • 🟢 misc
+-  modulo • 🟢 mod
+-  multiplier • 🟢 mplr
+-  mutable • 🟡 mut { variables }
 
 ### N
 
--  🔴 n • no {only with **yes**}
--  🟢 nav • navigation
--  🟡 ne • not equal {binary operator}
--  🟢 net • network
--  🟡 nl • newline {endcoding}
--  🟢 num • number
+-  navigation • 🟢 nav
+-  network • 🟢 net
+-  newline • 🟡 nl { endcoding }
+-  no • 🟡 n { only with **yes** }
+-  node 🟡 $... • { selector / DOM }
+-  not equal • 🟡 ne { binary operators }
+-  number • 🟢 num
 
 ### O
 
--  🟢 obj • object
--  🔴 ord • order {data science}
--  🟡 op • operation {binary operation}
--  🟢 opt • option / optional
--  🟢 os • operating system
--  🟢 out • output
+-  object • 🟢 obj
+-  object key • 🟡 key { only with **val** }
+-  operation • 🟡 op { binary operations }
+-  optional • 🟢 opt
+-  option • 🟢 opt
+-  operating system • 🟢 os
+-  order • 🟡 ord { data science }
+-  output • 🟢 out
 
 ### P
 
--  🟡 p • pointer {memory}
--  🟢 param • parameter
--  🟢 perf • performance
--  🟢 pic • picture
--  🟢 pos • position
--  🟢 pred • prediction
--  🟢 pref • preference
--  🟢 prev • previous
--  🟡 proc • process {thread}
--  🟢 prod • production
--  🟢 prof • profiler
--  🟢 ptr • pointer
--  🟢 pr • pull request
--  🟡 pwr • power {energy}
--  🟢 px • pixel
+-  parameter • 🟢 param
+-  performance • 🟢 perf
+-  picture • 🟢 pic
+-  pixel • 🟢 px
+-  pointer
+   -  🟡 p { memory }
+   -  🟢 ptr
+-  position • 🟢 pos
+-  prediction • 🟢 pred
+-  preference • 🟢 pref
+-  previous • 🟢 prev
+-  process • 🟡 proc { threads }
+-  production • 🟢 prod
+-  profiler • 🟢 prof
+-  power • 🟡 pwr { energy }
 
 ### Q
 
--  🟢 q / qry • query
+-  query
+   -  🟢 q
+   -  🟢 qry
 
 ### R
 
--  🟡 r • radius {circle / shape}
--  🟢 rand • random
--  🟢 rec • record
--  🟡 rect • rectangle {shape}
--  🟢 recv • receive
--  🟢 ref • reference
--  🟢 regex / rgx • regular expression
--  🟢 rem • remove
--  🟢 repo • repository
--  🟢 req • required / requested
--  🟢 res • response / result
--  🟢 ret • return
--  🟢 rev • revision
--  🟢 rng • range
+-  radius • 🟡 r { circles / shapes }
+-  random • 🟢 rand
+-  range • 🟢 rng
+-  receive • 🟢 recv
+-  record • 🟢 rec
+-  rectangle • 🟡 rect { shapes }
+-  reference • 🟢 ref
+-  regular expression
+   -  🟢 regex
+   -  🔴 rgx
```

Comment:
Can we still figure out something for this kind of cases?

I think it doesn't look that good if have only a few lines that break the nice pattern. I still vote for inlining. The worst case I can image in the practice is 5 abbrevs: 1 green, 2 yellow, 2 red.

So:
word • 🟢 w1 • 🟡 w2 { context } • 🟡 w3 { context } • 🔴 w4 • 🔴 w4

Or to stick with this specific case:
regular expression • 🟢 regex • 🔴 rgx

Reply:
we can wrap it in inline when there are maximum 3 abbrs.

Reply:
Now we have 3 abbrs in one place and 2 abbrs on a few places.  So max 3 works now, and we can figure it out later when there will be more abbrs somewhere.